flake.nix

```diff
@@ -0,0 +1,25 @@
+{
+  inputs.nixpkgs.url = "github:NixOS/nixpkgs/master";
+
+  outputs =
+    { self
+    , nixpkgs
+    , flake-utils
+    ,
+    }:
+    flake-utils.lib.eachDefaultSystem
+      (system:
+      let
+        pkgs = import nixpkgs {
+          inherit system;
+        };
+      in
+      {
+        devShells.default = pkgs.mkShell {
+          buildInputs = [
```

Comment:
I believe `nativeBuildInputs` should be preferred, for shell completions at least.